subprojects/architecture-test/src/changes/archunit_store/public_api_methods_do_not_reference_internal_types_as_parameters.txt

```diff
@@ -0,0 +1,380 @@
+Method <org.gradle.api.DefaultTask.configure(groovy.lang.Closure)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (DefaultTask.java:0)
+Method <org.gradle.api.DefaultTask.configure(groovy.lang.Closure)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (DefaultTask.java:0)
+Method <org.gradle.api.DefaultTask.doFirst(groovy.lang.Closure)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (DefaultTask.java:0)
+Method <org.gradle.api.DefaultTask.doLast(groovy.lang.Closure)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (DefaultTask.java:0)
+Method <org.gradle.api.DefaultTask.getInputs()> has arguments/return type org.gradle.api.internal.TaskInputsInternal that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (DefaultTask.java:0)
+Method <org.gradle.api.DefaultTask.getOutputs()> has arguments/return type org.gradle.api.internal.TaskOutputsInternal that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (DefaultTask.java:0)
```

Comment:
@wolfs Why didn't these show up as failures before?

Reply:
The name of the method changed, so you'd need to update `stored.rules` to point to the old location `public-api-methods-return-allowed-types.txt`.

Reply:
Aha, thanks.